week03/css/home.css

```
@@ -0,0 +1,183 @@
+@font-face {
+  font-family: '국립박물관문화재단클래식B';
+  src: url('https://cdn.jsdelivr.net/gh/projectnoonnu/noonfonts_twelve@1.0/국립박물관문화재단클래식B.woff') format('woff');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: '국립박물관문화재단클래식M';
+  src: url('https://cdn.jsdelivr.net/gh/projectnoonnu/noonfonts_twelve@1.0/국립박물관문화재단클래식M.woff') format('woff');
+  font-weight: normal;
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+#header {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  padding: 10px 15px;
+  background-color: white;
+  color: #6a75ca;
+  font-size: 18px;
+  border-bottom: 3px solid #B4C7E7;
+}
+
+body {
+  background-color:#E6EEFB;
+}
+
+.hello {
+  font-family: '국립박물관문화재단클래식B';
+  font-size: 3em;
+  color: black;
+  text-align: left;
+}
+
+#job {
+  color: #0D51FE;
+}
+
+.gabin {
+  font-family: '국립박물관문화재단클래식B';
+  font-size: 4em;
+  color: #0D51FE;
+  text-align: left;
+} 
+
+.intro {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  height: 100vh;
+  margin: 0 auto;
+  max-width: 1080px;
+}
+
+.intro img {
+  height: 23em;
+  display: block;
+  margin: auto;
+}
+
+.intro .content {
+  flex: 1; 
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+}
+
+.navbar {
+  background-color: white;
+  height: 75px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+
+}
+
+.nav1 {
+  float: left;
+  width: 20%;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.logo img {
+  width: 170px;
+  height: 80px;
+}
+
+.nav2 {
+  float: left;
+  width: 50%;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.nav3 {
+  float: left;
+  width: 40%;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+}
+
+.navbar a {
+  text-decoration: none;
+  color: gray;
+  font-weight: lighter;
+  font-size: 23px;
+}
+.navbar a:hover {
+  color: #0D51FE;
+}
+
+.navbar ul {
+  float: right;
+}
+
+.navbar li {
+  margin-left: 15px;
+  list-style-type: none;
+  float: left;
+  font-family: '국립박물관문화재단클래식B';
+}
+
+.navbar li:hover {
+  border-bottom: 2px solid #0D51FE;
+}

```

1. 클래스 이름뿐만 아니라, 부모 요소 이름도 적기
   - 계층 구조를 명확히 나타낼 수 있습니다.
2. `justify-content: space-between;`을 사용하여 자식 요소 사이에 간격을 추가합니다.
   - nav2 없이 로고 이미지와 네비게이션을 양쪽 정렬할 수 있습니다.
   - Contant의 줄바꿈 문제가 해결됩니다.

```suggestion
body header {
  height: 70px;
  padding: 15px 30px;

  position: fixed;
  top: 0; left: 0; right: 0;

  color: #6a75ca;
  font-size: 18px;
  background-color: white;
  border-bottom: 3px solid #B4C7E7;

  display: flex;
  flex-direction: row;
  justify-content: space-between;
  align-items: center;
}
body header a {}
body header a img {
  width: 170px;
  height: 80px;
}
body header nav {}
body header nav ul {
  float: right;
}
body header nav ul li {
  margin-left: 15px;
  list-style-type: none;
  float: left;
  font-family: '국립박물관문화재단클래식B';
}
body header nav ul li a {
  text-decoration: none;
  color: gray;
  font-size: 23px;
  font-weight: lighter;
}
body header nav ul li a:hover {
  color: #0D51FE !important;
  border-bottom: 2px solid #0D51FE;
}
```